src/credentials/tests/TestFabricTable.cpp

```diff
@@ -3247,6 +3300,378 @@ TEST_F(TestFabricTable, VidVerificationSigningWorksWithoutVvs)
     }
 }
 
+TEST_F(TestFabricTable, SettingVvsFailsWithIcacAndSucceedWithout)
```

Comment:
Test is actually testing that empty VVSC fails with ICAC

Reply:
Suggest also adding a test that VVS succeeds if the empty VVCS is omitted

Reply:
Done